src/main/java/com/zaxxer/hikari/HikariConfig.java

```diff
@@ -848,7 +867,7 @@ private void logConfiguration()
                value = dsProps;
             }
             value = (prop.contains("password") ? "<masked>" : value);
-            LOGGER.debug((prop + "................................................").substring(0, 32) + (value != null ? value : ""));
+            LOGGER.debug((prop + "................................................").substring(0, 32) + value);
```

Comment:
yes you can enhance it like that. however string properties are 'trimmed' using new utility method.